src/desktop/components/artwork_save/view.coffee

```diff
@@ -28,7 +28,8 @@ module.exports = class ArtworkSaveView extends Backbone.View
     if not @user.isLoggedIn()
       return mediator.trigger 'open:auth',
         width: '500px',
-        mode: 'register'
+        # mode: 'register'
+        mode: 'signup'
```

Comment:
I left this in as kind of a reminder or prompt for discussion that we need go through and consolidated/rename to use `signup` over `register`  as well as use `type` instead of `mode`.

Perhaps this can be done in a separate PR. I'm also wondering how we want to handle this seeing as we want to merge this auth-system branch into master and making the change from `mode` to `type` would be breaking (or a bigger refactor).

Reply:
👍 Usually a TODO is a better marker but I agree to achieving that consistency in a separate PR.